2-structured-data/config.py

```diff
@@ -51,15 +51,19 @@
 #   $ cloud_sql_proxy -instances=your-connection-name=tcp:3306
 #
 # Alternatively, you could use a local MySQL instance for testing.
+# Port 3306 is the standard MySQL port. If you need to use a different port,
+# change the 3306 to a different port number.
```

Comment:
This comment should only apply to the section about the Cloud SQL proxy, because they specify the port in the proxy startup command.